src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -754,7 +754,7 @@ protected function dumpContainer(ConfigCache $cache, ContainerBuilder $container
         $dumper = new PhpDumper($container);
 
         if (class_exists('ProxyManager\Configuration') && class_exists('Symfony\Bridge\ProxyManager\LazyProxy\PhpDumper\ProxyDumper')) {
-            $dumper->setProxyDumper(new ProxyDumper(md5($cache->getPath())));
+            $dumper->setProxyDumper(new ProxyDumper(substr(hash('sha256', $cache->getPath()), -7)));
```

Comment:
technically, because in this case, the has returns as hexadecimal instead of b64-encoded, meaning it's free from the trailing padding issue with b64. But that's too specific, now changed.